corehq/ex-submodules/casexml/apps/case/templates/case/partials/case_hierarchy.html

```diff
@@ -37,7 +36,9 @@
 </style>
 
 <script>
-    $(function () {
+    $.when(
```

Comment:
Weird. Is this required because this isn't wrapped in `{% block js-inline %}` and the `script` tag isn't wratpped in a `{% block js %}`? seems strange that you would have to fetch the javascript async, unless I've misunderstood.